databases/week1/dbweek1.sql

```diff
@@ -0,0 +1,111 @@
+-- Find out how many tasks are in the task table
+SELECT
+    COUNT(id)
```

Comment:
COUNT(id) as task_amount
_Pointer:_
Provide  a column name for columns you have created